python/cudf/cudf/tests/test_parquet.py

```diff
@@ -2545,6 +2545,10 @@ def normalized_equals(value1, value2):
         value1 = None
     if value2 is pd.NA or value2 is pd.NaT:
         value2 = None
+    if isinstance(value1, np.datetime64):
```

Comment:
pyArrow `timestamps` may sometime be `np.datetime64` dtype breaking this assert without this change.